edgelet/contrib/config/windows/config.yaml

```diff
@@ -111,7 +111,7 @@ agent:
   type: "docker"
   env: {}
   config:
-    image: "mcr.microsoft.com/azureiotedge-agent:1.0.8.1"
+    image: "mcr.microsoft.com/azureiotedge-agent:1.0.8.2"
```

Comment:
why have this change?

Reply:
This is an error in the diff. This change is already present in 1.0.8. I'm not sure why it is showing here.